packages/client/src/views/SysAdmin/Performance/WorkflowStatus.tsx

```diff
@@ -620,7 +630,15 @@ function WorkflowStatusComponent(props: WorkflowStatusProps) {
     ).map((row, idx) => {
       return {
         ...row,
-        id: <LinkButton>{row.id}</LinkButton>,
+        id: (
+          <LinkButton
+            onClick={() =>
+              props.goToSearchResult(row.id as string, 'tracking-id')
```

Comment:
@favour234 so yea, can we make this go directly to the declaration view? so not through search?

Reply:
Yes the page is called Record Audit. 
Any tracking ID link in the app should navigate directly to the Record Audit page

Reply:
correct..would update that.